components/device-mgt/org.wso2.carbon.device.mgt.core/src/main/java/org/wso2/carbon/device/mgt/core/dao/impl/ApplicationMappingDAOImpl.java

```diff
@@ -88,11 +88,11 @@ public int addApplicationMapping(int deviceId, int applicationId,
             }
             stmt.executeBatch();
 
-            rs = stmt.getGeneratedKeys();
-            while (rs.next()) {
-                mappingIds.add(rs.getInt(1));
-            }
-            return mappingIds;
+//            rs = stmt.getGeneratedKeys();
```

Comment:
Remove commented code.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/commit/77c4c3485830cda733fff66fbf472f79d38df45c